src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball(new Random().nextInt(100), new ColorSupplier().getRandomColor());

```

Comment:
add a constant for 100 - avoid magic numbers
make Random and ColorSupplier class-level fields - to avoid creating a new instance each time this method is invoked